Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.h

```diff
@@ -68,6 +68,8 @@ typedef NS_ENUM(NSUInteger, OWSMessageGestureLocation) {
 
 @property (nonatomic, weak) id<OWSMessageBubbleViewDelegate> delegate;
 
+@property (nonatomic) BOOL shouldSuppressHeaderFooterAndRounding;
```

Comment:
I find that the "mode flag" pattern often comes back to bite, especially as they start to interact.

What about creating a new ConversationViewItem, and setting its `isFirstInCluster` and `isLastInCluster` to true?

Reply:
Sorry - I meant to say that this is TIOLI feedback. It might not be worth changing.